css/main.css

```diff
@@ -0,0 +1,47 @@
+@media (min-width: 768px) {
+  .container {
+    max-width: 730px;
+  }
+}
+
+body {
+
+}
+
+.jumbotron h1 {
+    font-size: 2.8em;
+}
+
+.jumbotron p {
+    font-size: 1.3em;
+}
+
+div .row h3 {
+    font-size: 1.5em;
+}
+
+.thumbnail img {
+	padding-top: 10px;	
+}
+
+.page-header h1 {
+	font-size: 1.8em;
+}
+
+.btn btn-primary {
```

Comment:
While its *good* that you've applied these "override" styles in a separate file, its not *generally* considered to be a good idea to apply additional styles to the Bootstrap Classes. see: http://stackoverflow.com/questions/8084964/how-to-overwrite-styling-in-twitter-bootstrap
The preferred approach is to add your own custom class and apply styles to that class.